README.md

```diff
@@ -199,6 +199,174 @@ An error will be raised:
 
 `ActiveModel::ValidationError: Validation failed: Title can't be blank`
 
+#### Content Areas
```

Comment:
👏 thanks for the thorough docs! I'm starting to wonder if we might want to explore other avenues besides the README for documentation. What do you think?

Reply:
Yeah I think we need some more pages. I'd focus the Readme on simple usage. Detailed examples for ContentAreas and Motivation and design docs in their own pages? Feels like a separate PR.

Reply:
Agreed. Just wanted to know what you thought ❤️